packages/plugin-avatar/test/integration/spec/avatar.js

```diff
@@ -0,0 +1,81 @@
+/**!
+ *
+ * Copyright (c) 2015-2016 Cisco Systems, Inc. See LICENSE file.
+ */
+
+import {assert} from '@ciscospark/test-helper-chai';
+import sinon from '@ciscospark/test-helper-sinon';
+import fh from '@ciscospark/test-helper-file';
+import CiscoSpark from '@ciscospark/spark-core';
+import testUsers from '@ciscospark/test-helper-test-users';
+
+describe(`plugin-avatar`, () => {
+
+  let mccoy, spark, spock;
+
+  before(() => testUsers.create({count: 3})
```

Comment:
Looks like we're making three test users, but only using two.  Any reason we can't just make 2 test users?